src/language/blockString.ts

```diff
@@ -93,8 +93,8 @@ export function getBlockStringIndentation(value: string): number {
  */
 export function printBlockString(
   value: string,
-  indentation?: string = '',
-  preferMultipleLines?: boolean = false,
+  indentation: string = '',
+  preferMultipleLines: boolean = false,
```

Comment:
Can you please extract it in PR against `master`?

Reply:
@IvanGoncharov moved to https://github.com/graphql/graphql-js/pull/2879